app/controllers/merchant/items_controller.rb

```diff
@@ -12,27 +12,26 @@ def destroy
     redirect_to "/merchant/items"
   end
 
-  def new 
-    @item = Item.new
+  def new
+    @item = Item.new 
   end
 
   def create
     merchant = Merchant.where("id = #{current_user.merchant.id}").first
```

Comment:
Not sure the .first is necessary in the when finding the merchant; I believe there should only be one Merchant in the database with that id

Reply:
I asked about this too, Chase. Alex explained that this query returns an array-like object, so, he's not calling on the first of the result of the AR query, he's calling it on the array to get the merchant.  Maybe we could use find instead of where to get just the object, but this works.